pennylane/ftqc/parametric_midmeasure.py

```diff
@@ -127,7 +170,9 @@ def func(x, y):
         See :ref:`here <mid_circuit_measurements_statistics>` for more details.
     """
 
-    # ToDo: if capture is enabled, create and bind primitive here and return primitive instead (subsequent PR)
+    if capture.enabled():
+        primitive = _create_parametrized_mid_measure_primitive()
+        return primitive.bind(wires, angle=angle, plane=plane, reset=reset, postselect=postselect)
```

Comment:
```suggestion
        return primitive.bind(wires, angle, plane=plane, reset=reset, postselect=postselect)
```

I was just testing this locally. If you make `angle` a positional argument instead of a keyword argument of `bind`, we can make it dynamic. In other words, we would be able to do

```python
dev = qml.device("null.qubit", wires=1)

@qjit
@qml.qnode(dev)
def circuit(angle):
    qml.ftqc.measure_arbitrary_basis(0, angle, "XY")
    ...
```

Reply:
Awesome! I'll update that :)

Reply:
Is there an established convention we should follow for the ordering of wires and params? Like, `RX` usually comes out as `RX[n_wires=1] angle wire`, and this comes out as `wire angle`. 

Should I switch them, or is this arbitrary?

Reply:
Right, I gather the convention in the PennyLane layer is that the signature should have `wires` as the last positional argument, e.g.

```python
foo(arg1, arg2, ..., wires, kwarg1=<default>, kwarg2=<default>, ...)
```

I don't think we generally follow this convention with JAX primitives, though. Moreover, in this case, `plane` has to be a kwarg because it comes in a string (and since JAX can't trace strings, it can't be a positional arg), and it's nice to have `angle` and `plane` adjacent to each other in the function signature. This would look really weird, for example:

```python
measure_in_basis_p.bind(angle, wire, plane=plane)
```

Reply:
I meant more how it turns out in the jaxpr, where the qfunc:

```
def qfunc():
    qml.RX(1.23, wires=0)
    qml.Rot(1.23, 2.34, 4.56, 2)
    measure_x(0)
    measure_arbitrary_basis(2, angle=1.2, plane="XY")
```

gives us this jaxpr (note the parameters at the end of each operation)

```
>>> jax.make_jaxpr(qfunc)()
{ lambda ; . let
    _:AbstractOperator() = RX[n_wires=1] 1.23 0
    _:AbstractOperator() = Rot[n_wires=1] 1.23 2.34 4.56 2
    _:i64[] = measure_in_basis[plane=XY postselect=None reset=False] 2 0.0
    _:i64[] = measure_in_basis[plane=XY postselect=None reset=False] 2 1.2
  in () }
  ```
  
This is slightly jarring to me, because it's not what I've seen otherwise (normally you can skim the right edge to see the operation wires). But I don't know if this is a meaningful/intentional pattern, or just an incidental one I've gotten used to.

Reply:
Okay - I switched them so the jaxpr looks consistent, but if it somehow ends up causing you trouble that isn't easily resolved, we'll open a tiny PR and put it back the way it was 😅

Reply:
Should be all good, thanks @lillian542!